lib/string.rb

```diff
@@ -2,4 +2,45 @@ class String
   def humanized_all_first_capitals
     self.humanize.split(' ').map{|w| w.capitalize}.join(' ')
   end
+  
+  def downcase_words
+    self.scan(/\b\w+\b/).map(&:downcase)
+  end
+  
+  # Methods below created to customly setup slugs from friendly_id gem
+  
+  def short_name
```

Comment:
yeah, that's a completely new chore I think - just wanted to add a note about it - we shouldn't start making more changes until some other customer request drives us towards this part of the code again I think.